### PR TITLE
Update a comment in Credential.cs to make it accurate

### LINKED ISSUE #1658: Question regarding PSCredential.GetNetworkCredential()
Is this comment really true?

``` csharp
// NetworkCredential constructor only accepts plain string password in CoreCLR
// Since user can already access the plain text password via PSCredential.GetNetworkCredential().Password,
// this change won't be a security issue for PS on CSS.
...
```

https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/Credential.cs#L277

What I mean is that this is a non-trivial security regression compared to WindowsPowerShell, right?

Regardless of whether **the author** can access passwords ...

The "security issue" is a question of whether the password is stored in memory indefinitely (with non-deterministic garbage collection) as a string _that another user could read._

In the case of CoreFx NetworkCredential, the answer is yes.

With the full Framework and SecureString, as of 4.0 the answer is no.

### PR DESCRIPTION
**Pull request by daxian-dbw**
I should have updated this comment a long time ago ... Fix #1658

### CODE REVIEW

File: src/System.Management.Automation/engine/Credential.cs [hunk 1]
```diff
@@ -273,9 +273,10 @@ public NetworkCredential GetNetworkCredential()
                 if (IsValidUserName(_userName, out user, out domain))
                 {
 #if CORECLR
-                    // NetworkCredential constructor only accepts plain string password in CoreCLR
-                    // Since user can already access the plain text password via PSCredential.GetNetworkCredential().Password,
-                    // this change won't be a security issue for PS on CSS.
+                    // NetworkCredential constructor only accepts plain string password in .NET Core.
```
Comment (PaulHigin) [inline src/System.Management.Automation/engine/Credential.cs]: Please add a "TODO: " to this comment so that we don't forget about this work item.

Comment (daxian-dbw) [inline src/System.Management.Automation/engine/Credential.cs]: Fixed.

**Comment by PaulHigin** [comment]
LGTM

**Comment by daxian-dbw** [comment]
@TravisEz13 I will just merge this PR since it's just comment change and also got approved by Paul.

**Pull request merged**